contrib/grpc-cmake/CMakeLists.txt

```diff
@@ -1427,6 +1032,16 @@ target_link_libraries(grpc++_unsecure
   grpc_unsecure
 )
 
+
+if (_gRPC_SSL_INCLUDE_DIR AND _gRPC_SSL_LIBRARIES)
+  add_library(libgrpc ALIAS grpc)
+  add_library(libgrpc++ ALIAS grpc++)
+else()
+  add_library(libgrpc ALIAS grpc_unsecure)
+  add_library(libgrpc++ ALIAS grpc++_unsecure)
```

Comment:
`libgrpc++` is what other targets should be linked with via `target_link_libraries`.